docs/reference/api/docker_remote_api.md

```diff
@@ -115,6 +115,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `POST /containers/create` now takes `StorageOpt` field.
 * `GET /info` now returns `SecurityOptions` field, showing if `apparmor`, `seccomp`, or `selinux` is supported.
 * `GET /networks` now supports filtering by `label`.
+* `POST /containers/create` now takes `MaximumIOps` and `MaximumIOBps` fields on a Windows daemon.
```

Comment:
I would add `only` to the end of this - my first reaction was "do you mean in addition to, or exclusive?" then I read on....